src/app/services/saved-item.service.ts

```diff
@@ -48,4 +48,35 @@ export class SavedItemService {
         })
       );
   }
+
+  findJikanItemExistanceInSavedItems(jikanId: number): Observable<SavedItem> {
```

Comment:
Typo `existance` -> `existence`
```suggestion
  findJikanItemExistenceInSavedItems(jikanId: number): Observable<SavedItem> {
```